OpenRA.Game/Platform.cs

```diff
@@ -94,12 +94,24 @@ static string GetSupportDir()
 			return dir + Path.DirectorySeparatorChar;
 		}
 
-		public static string GameDir { get { return AppDomain.CurrentDomain.BaseDirectory; } }
+		public static string GameDir { get { return gameDir.Value; } }
+		static Lazy<string> gameDir = Exts.Lazy(GetGameDir);
```

Comment:
Alright, I saw the other dir-properties having that so I guessed it wouldn't hurt having it on GameDir too